test/Etg.Yams.Core.Test/Storage/DeploymentConfigTest.cs

```diff
@@ -2,36 +2,40 @@
 using System.IO;
 using System.Linq;
 using Etg.Yams.Application;
+using Etg.Yams.Json;
 using Etg.Yams.Storage.Config;
 using Etg.Yams.Test.Utils;
+using Etg.Yams.Utils;
 using Xunit;
 using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
 
 namespace Etg.Yams.Test.Storage
 {
     public class DeploymentConfigTestFixture
```

Comment:
Tests added. Found some issues and ended up replacing the internal HashSet with a Dictionary which was more appropriate.